From ltsi-dev-bounces@lists.linuxfoundation.org Wed Oct 29 09:53:58 2014
From: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
Date: Wed, 29 Oct 2014 08:52:57 +0800
Subject: [LTSI-dev]  [PATCH 7/8] pwm: lpss: Fix build failure on PowerPC
To: LTSI Mailing List <ltsi-dev@lists.linuxfoundation.org>
Cc: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
Message-ID: <1414543978-24145-8-git-send-email-rebecca.swee.fun.chang@intel.com>


From: Thierry Reding <thierry.reding@gmail.com>

An x86 build seems to pull in the linux/io.h include indirectly. On
PowerPC that doesn't happen and the build breaks due to the readl() and
writel() functions not being declared. Fix this by explicitly including
linux/io.h.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Thierry Reding <thierry.reding@gmail.com>
(cherry picked from commit e0c86a3b63e948e51a47d17382c7cd8711d19750)

Signed-off-by: Chang Rebecca Swee Fun <rebecca.swee.fun.chang@intel.com>
---
 drivers/pwm/pwm-lpss.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/pwm/pwm-lpss.c
+++ b/drivers/pwm/pwm-lpss.c
@@ -13,6 +13,7 @@
  * published by the Free Software Foundation.
  */
 
+#include <linux/io.h>
 #include <linux/kernel.h>
 #include <linux/module.h>
 
